test/integrations/destinations/campaign_manager/dataDelivery/business.ts

```diff
@@ -0,0 +1,605 @@
+                      floodlightConfigurationId: '213123123',
+                      ordinal: '1',
+                      floodlightActivityId: '456543345245',
+                      value: 7,
+                      gclid: '123',
+                      limitAdTracking: true,
+                      childDirectedTreatment: true,
+                    },
+                    kind: 'dfareporting#conversionStatus',
+                  },
+                  {
+                    conversion: {
+                      timestampMicros: '1668624722000000',
+                      floodlightConfigurationId: '213123123',
+                      ordinal: '1',
+                      floodlightActivityId: '456543345245',
+                      value: 8,
+                      gclid: '321',
+                      limitAdTracking: true,
+                      childDirectedTreatment: true,
+                    },
+                    kind: 'dfareporting#conversionStatus',
+                  },
+                ],
+                kind: 'dfareporting#conversionsBatchInsertResponse',
+              },
+              status: 200,
+            },
+            response: [
+              {
+                statusCode: 200,
+                metadata: {
+                  jobId: 1,
+                  attemptNum: 1,
+                  userId: 'dummyUserId',
+                  sourceId: 'dummySourceId',
+                  destinationId: 'dummyDestinationId',
+                  workspaceId: 'dummyWorkspaceId',
+                  secret: {},
+                  dontBatch: false,
+                },
+                error: 'success',
+              },
+              {
+                statusCode: 200,
+                metadata: {
+                  jobId: 2,
+                  attemptNum: 1,
+                  userId: 'dummyUserId',
+                  sourceId: 'dummySourceId',
+                  destinationId: 'dummyDestinationId',
+                  workspaceId: 'dummyWorkspaceId',
+                  secret: {},
+                  dontBatch: false,
+                },
+                error: 'success',
+              },
+            ],
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'cm360_v1_scenario_2',
+    name: 'campaign_manager',
+    description:
+      '[Proxy v1 API] :: Test for a valid request - where the destination responds with 200 with error for conversion 2',
+    successCriteria: 'Should return 200 with partial failures within the response payload',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            ...commonRequestParameters,
+            endpoint:
+              'https://dfareporting.googleapis.com/test_url_for_invalid_request_conversion_2',
+          },
+          metadataArray,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
```

Comment:
Shouldn't the http response code remain same?

Reply:
For partial failure the response status is 200 but internal status of each response object represents the status of each job

Reply:
Whatever response code v0 returns, v1 also should return the same.

Reply:
As discussed this is consolidated in the following [PR](https://github.com/rudderlabs/rudder-transformer/pull/3049)